CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [2.6.5] - (20/01/21) - Test 1 - CIA Down Under AI iOS
+## [2.6.6] - (05/02/21) - Test 2 - CIA Down Under AI iOS
+
+### Bugs
+- Fix memory crash
 
```

Comment:
why are we replacing `2.6.5` with `2.6.6`, shouldn't the newer release be added on top ?

Reply:
Fixed